ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/ReportReviewForm.tsx

```diff
@@ -63,6 +63,25 @@ function ReportReviewForm({ formValues }: ReportReviewFormParams): ReactElement
             <li>None</li>
         );
 
+    const deliveryDestinations =
+        formValues.deliveryDestinations.length !== 0 ? (
+            formValues.deliveryDestinations.map((deliveryDestination) => (
+                <li key={deliveryDestination.notifier?.id}>{deliveryDestination.notifier?.name}</li>
+            ))
+        ) : (
+            <li>None</li>
+        );
+
+    const mailingLists =
+        formValues.deliveryDestinations.length !== 0 ? (
+            formValues.deliveryDestinations.map((deliveryDestination) => {
+                const emails = deliveryDestination.mailingLists.join(', ');
+                return <li key={emails}>{emails}</li>;
+            })
+        ) : (
+            <li>None</li>
+        );
```

Comment:
In the first implementation of VM Reports, we did not allow users to leave the email address blank. (In other words, they could not just let the default recipient of an email notifier be used.)

Are we allowing that in this iteration of Reports, or is this just to account for when there is no notifier selected, this says "None" as well?

Reply:
This would account for the case where they don't set any delivery destinations, which is possible according to the mocks and APIs